pubsub/google/cloud/pubsub_v1/subscriber/futures.py

```diff
@@ -39,16 +39,19 @@ def running(self):
 
         .. note::
 
-            A ``False`` value here does not necessarily mean that the
+            A :data:`False` value here does not necessarily mean that the
             subscription is closed; it merely means that _this_ future is
             not the future applicable to it.
 
             Since futures have a single result (or exception) and there is
-            not a concept of resetting them, a closing re-opening of a
+            not a concept of resetting them, a closing / re-opening of a
             subscription will therefore return a new future.
 
         Returns:
-            bool: ``True`` if this subscription is opened with this future,
-                ``False`` otherwise.
+            bool: :data:`True` if this subscription is opened with this
+            future, :data:`False` otherwise.
```

Comment:
@lukesneeringer Can you verify this **note** above?

Reply:
That note is still correct. However, with your change, a `True` value _should_ necessarily mean that the subscription is open.